dlclibrary/dlcmodelzoo/modelzoo_download.py

```diff
@@ -62,41 +66,58 @@ def download_huggingface_model(modelname, target_dir=".", remove_hf_folder=True)
         Directory where to store the model weights and pose_cfg.yaml file
     remove_hf_folder : bool, default True
         Whether to remove the directory structure provided by HuggingFace after downloading and decompressing data into DeepLabCut format.
+    rename_mapping : dict, default None
+        Dictionary to rename the downloaded file. If None, the original filename is used.
     """
     from huggingface_hub import hf_hub_download
     import tarfile
     from pathlib import Path
+    from ruamel.yaml.comments import CommentedBase
 
     neturls = _load_model_names()
     if modelname not in neturls:
         raise ValueError(f"`modelname` should be one of: {', '.join(modelname)}.")
```

Comment:
I suppose, there should be:

```python
if modelname not in neturls:
         raise ValueError(f"`modelname` should be one of: {', '.join(neturls)}.")
```

Otherwise raises wrong error:

`ValueError: `modelname` should be one of: s, u, p, e, r, a, n, i, m, a, l, _, t, o, p, v, i, e, w, m, o, u, s, e, _, h, r, n, e, t, w, 3, 2.
`